### PR TITLE
missed ad server

### PR DESCRIPTION
**Pull request by wout-codes**
Serves fake/scam notifications in top right corner.

### CODE REVIEW

File: EnglishFilter/sections/adservers.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 !
 !! https://a.bestcontentitem.top/warp/*?r=
 !
+||zopim.com^$third-party
```
Comment (zloyden) [inline EnglishFilter/sections/adservers.txt:8]: We have already it in the Annoyances filter.

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: @zloyden Don't make it third party.

Comment (zloyden) [reply EnglishFilter/sections/adservers.txt:8]: What do you mean? As I said earlier, it's in the Annoyances filter https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/sections/widgets.txt#L176

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: @zloyden Like this: `||zopim.com^$third-party` to this `||zopim.com^` without any domains.

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: Ok wait nevermind, it may cause false positives

Comment (zloyden) [reply EnglishFilter/sections/adservers.txt:8]: Yep, that's why we added the domains with `~`. In other words, these are exceptions.

Comment (wout-codes) [reply EnglishFilter/sections/adservers.txt:8]: Yes.

**Pull request closed without merging**